### DISCUSSION TITLE
Pagefind UI: debounce user input?

### DISCUSSION THREAD

**Discussion by dprothero** [Pagefind/pagefind#104] [Q&A] [answered]
Is there a way to have the UI debounce the user input? It's really distracting to have search results showing up for every key press.

In other words, don't have the search actually execute until something like 300ms after the last key press.

**Comment by bglw** [comment]
This definitely sounds like an option that could be added for the UI

**Reply by bglw** [reply]
Converted into issue #105

**Comment by bglw** [comment, marked as answer]
Released in v0.9.0 🎉
